docs/reference/cpp_language_extensions.rst

```diff
@@ -104,7 +104,7 @@ You can include your kernel arguments after these parameters.
 .. code-block:: cpp
```

Comment:
Please also remove this line 102 from documentation:

"You can include your kernel arguments after these parameters."